tests/e2e/run_qwen_coder1_grpo.sh

```diff
@@ -0,0 +1,45 @@
+set -x
+
+export VLLM_ATTENTION_BACKEND=XFORMERS
+
+huggingface-cli download Qwen/Qwen2.5-0.5B --local-dir $HOME/models/Qwen/Qwen2.5-0.5B
+
+python3 -m verl.trainer.main_ppo \
```

Comment:
what's the expected performance with this script? If it's reasonably good we should put a reference example under `example/` folder?